Makefile

```diff
@@ -92,15 +92,14 @@ EXTRADEPS  = $(if $(findstring priv,$*),$(PRIV_HEADERS_EXPANDED) $(PRIVDIR$(BITW
 %.elf: $$(SOURCEFILE) $$(EXTRADEPS)
 	riscv64-unknown-elf-gcc -g -o $@ -march=rv$(BITWIDTH)g$(CMPR_FLAGS)_zfa_zba_zbb_zbc_zbs_zfh_zicboz_zicbop_zicbom_zicond_zbkb_zbkx_zknd_zkne_zknh_zihintpause -mabi=$(MABI) -mcmodel=medany \
     -nostartfiles -I$(TESTDIR) -T$(TESTDIR)/link.ld $<
-	$(MAKE) $@.objdump $@.memfile
+	$(MAKE) $@.objdump
 
 %.elf.objdump: %.elf
 	riscv64-unknown-elf-objdump -S -D $< > $@
-# 	no need for extractFunctionRadix unless doing hard debugging, and it runs slowly
-#	extractFunctionRadix.sh $@
 
-%.elf.memfile: %.elf
-	riscv64-unknown-elf-elf2hex --bit-width $(BITWIDTH) --input $< --output $@
+#%.elf.memfile: %.elf
```

Comment:
For that matter I don't see a speedup from removing extractRadix either. `make --jobs` takes 30 seconds with or without extractRadix and it slows down simulation because it needs to run the Makefile.